### ISSUE TITLE
add iso19139 as an output encoding of ogc api-records

### ISSUE THREAD

**Issue by pvgenuchten** [geopython/pycsw#774]
a typical usecase for pycsw is to load iso records as input data and then dissiminate them via ogc-api-records, as a final step i would like to see the option to fetch the iso19139 record in an xml encoding, great to see this is already implemented as

http://localhost:8000/collections/metadata:main/items/demo?f=xml

Next step is to add the xml-output-option to the top right html template

![image](https://user-images.githubusercontent.com/299829/184098741-198c72e2-e67e-4eb3-8f49-8cb62c33c756.png)

Do we need some diversification on xml flavours, eg iso19115:2003, iso19115:2018, ATOM

**Comment by kalxas** [comment]
yes, this is an open discussion in OGC API Records SWG on how to actually use different flavours of XML

**Comment by pvgenuchten** [comment]
i added the xml output option in a pycsw-skin project, is this approach valid for the core project, i could prepare a PR

https://github.com/pvgenuchten/pycsw-skin/blob/e61eddb293461b4fe6e8a3505ba40a03e4c1f065/item.html#L15-L17
